<launch>


<node name="mission_node" pkg="dt_mission" type="dt_mission_node" output="screen">

<!--mission_point-->
    <rosparam file="$(find dt_mission)/config/mission_point.yaml" command="load"/>
    <rosparam file="$(find dt_mission)/config/uav_1_points.yaml" command="load"/>
    <rosparam file="$(find dt_mission)/config/uav_2_points.yaml" command="load"/>
    <rosparam file="$(find dt_mission)/config/uav_3_points.yaml" command="load"/>

    <param name="mission_point_err" value="0.05"/>
    <param name="mission_control_sub_topic" value="/dt_mission/start"/>
    <param name="mission_change_type" value="0"/>                 <!--0:Auto 1:Other-->
    <param name="feeback_sub_topic" value="/mavros/local_position/pose"/>
    <param name="next_mission_point_sub_topic" value="/dt_mission/mission_control/next"/>
    <param name="mission_point_pub_topic" value="/dt_mission/mission_point"/>
    <param name="yawl_file_name" value="mission_point"/>
    <param name="mission_1_yawl_file_name" value="uav_1_points"/>
    <param name="mission_2_yawl_file_name" value="uav_2_points"/>
    <param name="mission_3_yawl_file_name" value="uav_s_points"/>
    <param name="circle_run" value="0"/>                          <!--0:Not Circle Run. 1:Circle Run-->

    <param name="local_machine_local_position_pub_topic" value="/local_machine/target/local_position"/>
    <param name="local_machine_global_position_pub_topic" value="/local_machine/target/global_position"/>
    <param name="local_machine_velocity_pub_topic" value="/local_machine/target/velocity"/>
    <param name="local_machine_control_apply_pub_topic" value="/local_machine/control/apply"/>
    <param name="local_machine_target_cmd_type_pub_topic" value="/local_machine/target_cmd_type"/>
   <!--generate path-->
    <param name="init_x" value="0.0"/>
    <param name="init_y" value="0.0"/>
    <param name="init_z" value="0.0"/>
     <param name="init_velocity_x" value = "0.0"/>
     <param name="init_velocity_y" value = "0.0"/>
     <param name="init_velocity_z" value = "0.0"/>
     <param name="max_velocity_x" value = "1.5"/>
     <param name="max_velocity_y" value = "1.5"/>
     <param name="max_velocity_z" value = "1.5"/>
     <param name="max_acc_x" value = "3.0"/>
     <param name="max_acc_y" value = "3.0"/>
     <param name="max_acc_z" value = "3.0"/>
     <param name="max_jerk_x" value = "12.0"/>
     <param name="max_jerk_y" value = "12.0"/>
     <param name="max_jerK_z" value = "12.0"/>
     <param name="TargetPosSubTopic" value="/target_position"/>
     <param name="RefPosPubTopic" value ="/mavros/setpoint_position/local"/>
     <param name="curPositionSubTopic" value="/mavros/local_position/pose"/>
     <param name="curVelocitySubTopic" value="/mavros/local_position/velocity"/>
     <param name="UpdateHz" value="50"/>

  </node>
</launch>
